### PR TITLE
[DEV APPROVED] 7421 - BUG: Updating Publify API call for new blog url

### PR DESCRIPTION
**Pull request by johnplayer1982**
Relates to a publify PR to add functionality to create a JSON feed from the blog

[BUG: blog JSON feed for 'From our blog' section](https://github.com/moneyadviceservice/publify/pull/190)

### CODE REVIEW

**Comment by jongilbraith** [comment]
Probably good to just rebase out the two changes in `.env` but otherwise, 👍

**Comment by pmcostello** [comment]
👍 squash and merge :)

**Comment by johnplayer1982** [comment]
Squashed and env commits rebased out, thanks @pmcostello & @jongilbraith

**Comment by rnalexander** [comment]
👍

**Comment by jongilbraith** [comment]
👍

**Comment by misaka** [comment]
:+1: ... I worked on this with John, but thought I'd throw my thumb into it too ... ;)

**Comment by UmairIft** [comment]
I kind of know what's going on with this PR, so ... 👍

File: .env [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # N.B. This file is intended as a placeholder. Never put any real values in this file and commit.
 ####################################################################################################
 
-AZURE_ASSETS_STORAGE_CMS_ACCOUNT_CONTAINER='container'
-AZURE_ASSETS_STORAGE_CMS_ACCOUNT_KEY='key'
-AZURE_ASSETS_STORAGE_CMS_ACCOUNT_NAME='name'
+AZURE_ASSETS_STORAGE_CMS_ACCOUNT_CONTAINER='cms-dev'
+AZURE_ASSETS_STORAGE_CMS_ACCOUNT_KEY='T+NorXajeRo5pAHTr1lASLXYjO1Ufqj6MMjecSV6i0zm5lLsZejQbNGpW3+YHKOVRaJyhnUsjjCU1fw6t7ciOg=='
+AZURE_ASSETS_STORAGE_CMS_ACCOUNT_NAME='masassets'
```
Comment (johnplayer1982) [inline .env]: These will be removed before next review

**Comment by jongilbraith** [comment]
👍 

Like the `MAS.env` business 😄

**Comment by johnplayer1982** [comment]
This requires 1 more thumb, since the changes introduced in the last commit, related to:
https://github.com/moneyadviceservice/scripts/pull/656

**Comment by tomas-stefano** [comment]
:+1:

**Pull request merged**